concourse/scripts/cf_push_on_git_change.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
This is causing errors as the cf-cli image doesn't have bash.

~~~
$ docker run -ti governmentpaas/cf-cli sh -i
/ # /bin/bash
sh: /bin/bash: not found
~~~

Or in the context of a pipeline run. 

~~~
sh: ../paas-cf/concourse/scripts/cf_push_on_git_change.sh: not found
~~~